osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapCarousel.cs

```diff
@@ -748,6 +748,7 @@ private void createCarousel(Action<BeatmapCarousel> carouselAdjust = null, Conta
                 carousel = new TestBeatmapCarousel
                 {
                     RelativeSizeAxes = Axes.Both,
+                    Clock = Clock
```

Comment:
This seems really weird. Are you instead able to remove whatever is dependent on this into BDL or LoadComplete?